flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/functions/hive/HiveScalarFunction.java

```diff
@@ -104,4 +114,66 @@ public Object eval(Object... args) {
 	 * Evaluation logical, args will be wrapped when is a single array.
 	 */
 	protected abstract Object evalInternal(Object[] args);
+
+	private void setArguments(CallContext callContext) {
+		DataType[] inputTypes = callContext.getArgumentDataTypes().toArray(new DataType[0]);
+		Object[] constantArgs = new Object[inputTypes.length];
+		for (int i = 0; i < constantArgs.length; i++) {
+			if (callContext.isArgumentLiteral(i)) {
```

Comment:
We should also check for `isNullLiteral`?

Reply:
Looks like `getArgumentValue` will return empty for null literal. But we should add test for null literal.